options/options.html

```
@@ -294,6 +294,80 @@ <h3>介面設定</h3>
+               與既有「Notion 連接」card 完整分開，禁止共用 DOM 或 storage。
+               此整個 div 由 options.js 的 initAccountUI() 負責管理。 -->
+          <div id="account-card" class="card" style="display: none">
+            <h3>帳號登入</h3>
+
+            <!-- 未登入狀態（預設顯示） -->
+            <div id="account-logged-out">
+              <p class="section-desc" style="margin-bottom: 12px">
+                使用 Google 帳號登入，享受同步與進階功能。
+              </p>
+              <div class="auth-actions">
+                <button id="account-login-button" class="btn-primary">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)